src/domUpdates.js

```diff
@@ -0,0 +1,452 @@
+// IMPORTS
```

Comment:
This was the definition of a doozy! After checking in with Hannah, we think we're at least on the right track here. And hey- project still functions!